src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -72,18 +76,16 @@ class PropertyAccessor implements PropertyAccessorInterface
      * Should not be used by application code. Use
      * {@link PropertyAccess::createPropertyAccessor()} instead.
      *
-     * @param int $magicMethods A bitwise combination of the MAGIC_* constants
-     *                          to specify the allowed magic methods (__get, __set, __call)
-     *                          or self::DISALLOW_MAGIC_METHODS for none
-     * @param int $throw        A bitwise combination of the THROW_* constants
-     *                          to specify when exceptions should be thrown
+     * @param int $throw A bitwise combination of the THROW_* constants
```

Comment:
Why removing the `magicMethods` documentation?
And I'd added the documentation about `byPassHooksOnProperty` as well

Reply:
I should have been more careful, psalm removed it because the property is `$magicMethodsFlags` and not `$magicMethods`